apps/cyberstorm-storybook/.storybook/preview.js

```diff
@@ -30,7 +36,11 @@ export const decorators = [
           <SessionProvider>
             <DapperProvider dapperConstructor={() => new DapperFake()}>
               <LinkingProvider value={LinkLibrary}>
-                <Story />
+                <div className={styles.root}>
+                  <Header />
```

Comment:
I don't quite see why we'd want to show header/footer on every showcase. Doesn't it just introduce noise making it hard to notice issues on the component itself? It also triggers changes for every Story everytime the header or footer is changed.

Reply:
True, but we do want to mirror the NextJS layouts to storybook. It's a work in progress.